spec/monitoring_spec.ts

```diff
@@ -402,24 +402,27 @@ describe('monitoring', () => {
             .perform();
         authoring.save();
         authoring.close();
+        expect(monitoring.getAllItems().count()).toBe(3);
         el(['content-profile-dropdown']).click();
-        var items = els(['content-profiles']);
-
-        browser.wait(ECE.hasElementCount(items, 2));
-        items.get(1).click();
+        browser.wait(ECE.hasElementCount(els(['content-profiles']), 2));
+        el(['content-profile-dropdown'], by.buttonText('Simple')).click();
+        expect(monitoring.getAllItems().count()).toBe(1);
         expect(monitoring.getTextItemBySlugline(0, 0)).toBe('STORY1 SLUGLINE');
```

Comment:
Looks like this story is always there. It'd be more useful to check the presence of an item which is supposed to be filtered out.